server/controllers/admin.py

```diff
@@ -78,7 +78,7 @@ def get_courses(cid=None):
     matching_courses = [c for c in courses if c.id == cid]
     if len(matching_courses) == 0:
         abort(401)
-    current_course = matching_courses[0]
+    current_course = max(matching_courses, key=lambda course: course.created)
```

Comment:
This is fine - but adding an order_by in the enrollment query seems like the right place for this